api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/ListTranslatorImpl.java

```diff
@@ -48,6 +54,8 @@ public ListResource toFhirResource(Cohort cohort) {
 			list.setStatus(ListResource.ListStatus.RETIRED);
 		}
 		
+		list.setEntry(listEntryTranslator.toFhirResource(cohort));
```

Comment:
It's preferred to use `list.addEntry(listEntryComponent);`

Reply:
So your listEntryTranslator should return **listEntryComponent** instead of **a list of listEntryComponent**
modify your listEntryComponentTranslator::toFhirResource
```java
ListResource.ListEntryComponent entry = new ListResource.ListEntryComponent();
for (CohortMembership member : cohort.getMemberships()) {
	if (!member.getVoided() && (member.getEndDate() == null || member.getEndDate().after(new Date()))) {			 
                 entry.setItem(patientReferenceTranslator.toFhirResource(patientDao.getPatientById(member.getPatientId())));
		 entry.setDate(member.getStartDate());
		}
	}
return entry;
```

Reply:
I'm already returning a list after adding list entries so in this case list.setEntry() is the suitable method to use. The reason for returning a list is because I don't have an option of passing CohortMember to the translator since it doesn't exist in 2.0.5. This means I can only pass Cohort  and process all the items in one place hence returning a list